tests/integration/__init__.py

```diff
@@ -574,9 +592,9 @@ def use_cluster(cluster_name, nodes, ipformat=None, start=True, workloads=None,
 
         # This will enable the Mirroring query handler which will echo our custom payload k,v pairs back
 
-        if 'graph' not in workloads:
-            if PROTOCOL_VERSION >= 4:
-                jvm_args = [" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"]
+        #if 'graph' not in workloads:
```

Comment:
Why? It seems that you do try to preserve the compatibility with Cassandra (otherwise why do you need this `isScylla` thing?